go-backend/service/recognition_hi5_http_test.go

```diff
@@ -0,0 +1,102 @@
+package service
+
+import (
+	"errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/suite"
+	"joshsoftware/peerly/db"
+	"net/http"
+)
+
+type RecognitionHi5HandlerTestSuite struct {
+	suite.Suite
+	dbMock *db.DBMockStore
+}
+
+func (suite *RecognitionHi5HandlerTestSuite) SetupTest() {
+	suite.dbMock = &db.DBMockStore{}
+}
+
```

Comment:
I am seeing that we are using `mock.Anything` for each parameter while mocking requests. It's a bad practice. Ideally we should be using the actual parameter that is supposed to be passed to the function.

mock.Anything should only be used in cases where you can predict the value before calling the function. eg: some time value